esp-hal/ld/esp32c2/esp32c2.x

```diff
@@ -47,30 +47,38 @@ SECTIONS {
 INSERT BEFORE .rwtext;
 
 SECTIONS {
-  /**
-   * This dummy section represents the .text section but in rodata.
-   * Thus, it must have its alignement and (at least) its size.
-   */
-  .text_dummy (NOLOAD):
+  .rotext_dummy (NOLOAD) :
   {
-    /* Start at the same alignement constraint than .text */
-    . = ALIGN(4);
+    /* This dummy section represents the .rodata section but in ROTEXT.
```

Comment:
I still don't understand why these dummies are needed. It would be so nice to have an explanation

Reply:
let me see how to come up with something understandable 👍